src/mainview.cpp

```diff
@@ -12,6 +13,11 @@ using namespace ui;
 CMainView::CMainView( QWidget *pParent ) :
 	QDialog( pParent )
 {
+	CFileSystemSearchProvider provider;
+	char *installDir = new char[1048];
```

Comment:
Is there any reason why this needs to be a pointer?

Reply:
Because it needs to get the contents from GetAppInstallDir.
`char installDir[1048];`
Is not allowed in some compilers.

Reply:
Better question: Why is it not 1024?

Reply:
SAPP's max path is 1048, it's what Momentum Mod used when I extracted the code I needed from their Open Source version.